carmin.yaml

```diff
@@ -86,6 +86,24 @@ paths:
           schema:
             $ref: '#/definitions/errorCodeAndMessage'
 
+  /pipelines:
+    get:
+      description: Returns a list of available pipelines
```

Comment:
We could use the full description as in the CARMIN specification: List pipelines. All the pipelines that the user can execute must be returned. It is up to the platform to return pipelines that the user cannot execute.  When studyIdentifier is present, all the pipelines that the user can execute in the study must be returned. In this case, execution rights denote the rights to execute the pipeline in the study.